src/libraries/System.Formats.Asn1/src/System/Formats/Asn1/AsnContentException.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.Serialization;
+
+namespace System.Formats.Asn1
+{
+    [Serializable]
+    public class AsnContentException : Exception
+    {
+        public AsnContentException()
+            : base(SR.ContentException_DefaultMessage)
+        {
+        }
+
+        public AsnContentException(string message)
+            : base(message ?? SR.ContentException_DefaultMessage)
+        {
+        }
+
+        public AsnContentException(string message, Exception inner)
+            : base(message ?? SR.ContentException_DefaultMessage, inner)
+        {
+        }
+
+        protected AsnContentException(SerializationInfo info, StreamingContext context)
+            : base(info, context)
+        {
+        }
+    }
+}
```

Comment:
The project is marked as nullable enabled.  Have all APIs been appropriately annotated?

Reply:
> Have all APIs been appropriately annotated?

I believe so.  The ctor was wrong because adding the message fallback was last minute, after remembering how frustrating it is when exceptions don't do the "allow null in a more complex overload to represent the default from a simpler overload" thing for messages.